src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -174,13 +180,13 @@ def upload_append_blob():
             'container_name': container_name,
             'blob_name': blob_name,
             'file_path': file_path,
-            'progress_callback': _update_progress,
+            'progress_callback': get_update_progress_fn(cmd),
             'maxsize_condition': maxsize_condition,
             'lease_id': lease_id,
             'timeout': timeout
         }
 
-        if supported_api_version(ResourceType.DATA_STORAGE, min_api='2016-05-31'):
+        if cmd.supported_api_version(ResourceType.DATA_STORAGE, min_api='2016-05-31'):
```

Comment:
I wouldn't expect that you would need to specify the ResourceType.

Reply:
will update.